package.json

```
@@ -1,5 +1,6 @@
 {
-  "name": "prep-template",
+  "name": "weather-app",
+  "contributors": ["Hemansh Khaneja", "Swatilekha Roy","Lucas Vinzon", "Tanmay Vyas", "Dat Quach", "Gavish Poddar", "Will Russel", "Arsalan Khattak", "Shivam Tewari", "Athar Mohammad", "Devansh Singh", "Harshal Raikwar"],
```

Just a little nitpick: add a space before `"Lucas Vinzon"`. Also correct Will's name.
```suggestion
  "contributors": ["Hemansh Khaneja", "Swatilekha Roy", "Lucas Vinzon", "Tanmay Vyas", "Dat Quach", "Gavish Poddar", "Will Russell", "Arsalan Khattak", "Shivam Tewari", "Athar Mohammad", "Devansh Singh", "Harshal Raikwar"],
```